mw-webapp/src/hooks/useLoad.ts

```diff
@@ -0,0 +1,76 @@
+import {useEffect, useState} from "react";
+
+/**
+ * Use Load hook Props
+ */
+interface useLoadProps<DataType, DependenciesType> {
```

Comment:
Is it conventional for us to use the postfix type? Could it improve readability in current case?

Reply:
Removed postfix